modules/reference/pages/public-metrics-reference.adoc

```diff
@@ -969,182 +969,304 @@ The number of transform processors in a specific state (running, inactive, error
 
-Bytes of memory used for spilled manifests currently cached in memory.
+*Type*: gauge
+
+=== redpanda_cloud_storage_cache_op_hit
+
+Number of get requests for objects that are already in cache.
+
+*Type*: counter
+
+=== redpanda_cloud_storage_cache_op_in_progress_files
+
+Number of files that are being put to cache.
+
+*Type*: gauge
+
+=== redpanda_cloud_storage_cache_op_miss
+
+Number of failed get requests because of missing object in the cache.
+
+*Type*: counter
 
 === redpanda_cloud_storage_cache_op_put
 
 Number of objects written into cache.
 
-=== redpanda_cloud_storage_segments
+*Type*: counter
 
-Total number of accounted topic segments in the cloud.
+=== redpanda_cloud_storage_cache_space_files
 
-=== redpanda_cloud_storage_jobs_local_segment_reuploads
+Number of objects in cache.
 
-Number of segment reuploads from local data directory.
+*Type*: gauge
 
-=== redpanda_cloud_storage_cache_trim_failed_trims
+=== redpanda_cloud_storage_cache_space_hwm_files
+
+High watermark of number of objects in cache.
+
+*Type*: gauge
+
+=== redpanda_cloud_storage_cache_space_hwm_size_bytes
+
+High watermark of sum of size of cached objects.
+
+*Type*: gauge
+
+=== redpanda_cloud_storage_cache_space_size_bytes
 
-Number of times Redpanda could not free the expected amount of space, indicating possible bug or configuration issue.
+Sum of size of cached objects.
+
+*Type*: gauge
+
+=== redpanda_cloud_storage_cache_space_tracker_size
+
+Number of entries in cache access tracker.
+
+*Type*: gauge
+
+=== redpanda_cloud_storage_cache_space_tracker_syncs
+
+Number of times the access tracker was updated with cache disk data.
+
+*Type*: counter
+
+=== redpanda_cloud_storage_cache_trim_carryover_trims
+
+Number of times we invoked carryover trim.
+
+*Type*: counter
 
 === redpanda_cloud_storage_cache_trim_exhaustive_trims
 
-Number of times a fast trim could not free enough space and had to fall back to a slower exhaustive trim.
+Number of times we couldn't free enough space with a fast trim and had to fall back to a slower exhaustive trim.
+
+*Type*: counter
+
+=== redpanda_cloud_storage_cache_trim_failed_trims
+
+Number of times could not free the expected amount of space, indicating possible bug or configuration issue.
+
+*Type*: counter
+
+=== redpanda_cloud_storage_cache_trim_fast_trims
+
+Number of times we have trimmed the cache using the normal (fast) mode.
+
+*Type*: counter
+
+=== redpanda_cloud_storage_cache_trim_in_mem_trims
+
+Number of times we trimmed the cache using the in-memory access tracker.
+
+*Type*: counter
+
+=== redpanda_cloud_storage_cloud_log_size
+
+Total size in bytes of the user-visible log for the topic.
+
+*Type*: gauge
 
 === redpanda_cloud_storage_deleted_segments
 
-Count of deleted remote segments.
+Number of segments that have been deleted from S3 for the topic. This may grow due to retention or non compacted segments being replaced with their compacted equivalent.
```

Comment:
```suggestion
Number of segments that have been deleted from S3 for the topic. This may grow due to retention or non-compacted segments being replaced with their compacted equivalent.
```